gsplat/strategy/default.py

```diff
@@ -72,7 +76,7 @@ class DefaultStrategy(Strategy):
 
     """
 
-    prune_opa: float = 0.005
+    prune_opa: float = 0.05
```

Comment:
Why change this? Does 2DGS use a different threshold? If so we should still keep the original one as default and set it to `0.05` in 2dgs training script.

Reply:
The original 2DGS implementation uses 0.05; using 0.005 will haven an excessive amount of gaussians.